src/routes/api/cryptor-migrations/migration-installation-verification.ts

```diff
@@ -0,0 +1,39 @@
+import { Request, Response } from "express";
+import { Installation } from "models/installation";
+
+export const CryptorMigrationInstallationVerificationPost = async (req: Request, res: Response): Promise<any> => {
+
+	const jiraHost: string = req.body?.jiraHost;
+
+	req.log = req.log.child({ operation: "migrate-installations-verification" });
```

Comment:
you're not using `req.log` in this route...

Reply:
fixed.